ship.config.js

```diff
@@ -7,6 +7,7 @@ const packages = [
   'packages/autocomplete-js',
   'packages/autocomplete-preset-algolia',
   'packages/autocomplete-plugin-recent-searches',
+  'packages/autocomplete-plugin-query-suggestions',
```

Comment:
FYI, Ship.js accepts `packagesToPublish: 'packages/*'`. But if you want to maintain an allow-list, then it's fine.